web/app/components/doc/tile.ts

```diff
@@ -0,0 +1,29 @@
+import Component from "@glimmer/component";
+
+interface DocTileComponentSignature {
+  Args: {
+    avatar?: string;
+    docID?: string;
+    docNumber?: string;
+    isOwner?: boolean;
+    isResult?: boolean;
+    modifiedAge?: string;
+    owner?: string;
+    productArea?: string;
+    snippet?: string;
+    status?: string;
+    thumbnail?: string;
+    title?: string;
+  };
+}
+
+export default class DocTileComponent extends Component<DocTileComponentSignature> {
+  protected get productAreaName(): string | undefined {
+    switch (this.args.productArea) {
+      case "Cloud Platform":
+        return "HCP";
+      default:
+        return this.args.productArea;
+    }
```

Comment:
This is definitely an improvement over the previous code, but longer-term we should think how we can make this more generic (might require some data provided by the backend) 🤔

Reply:
Yeah. Maybe an attribute like `flightIconName` or `shortName`.